src/Storages/Kafka/KafkaSettings.cpp

```diff
@@ -24,6 +24,7 @@ namespace ErrorCodes
     DECLARE(String, kafka_group_name, "", "Client group id string. All Kafka consumers sharing the same group.id belong to the same group.", 0) \
     /* those are mapped to format factory settings */ \
     DECLARE(String, kafka_format, "", "The message format for Kafka engine.", 0) \
+    DECLARE(String, kafka_format_avro_schema_registry_url, "", "For AvroConfluent format: Kafka schema registry URL", 0) \
```

Comment:
I don't think it's needed here. engine=Kafka accepts all usual input format settings